test/specs/lib/mergeThemes/mergeComponentStyles-test.ts

```diff
@@ -90,10 +91,10 @@ describe('mergeComponentStyles', () => {
 
     const merged = mergeComponentStyles(target, source)
 
-    const styleParam = {
+    const styleParam: ComponentStyleFunctionParam = {
       variables: { iconSize: 'large' },
       props: { primary: true },
-    }
+    } as any
```

Comment:
Why `any` is there? It doesn't match `ComponentStyleFunctionParam`?

Reply:
nope :( 
![screenshot 2018-11-14 at 14 14 46](https://user-images.githubusercontent.com/5442794/48484912-b4cc9b80-e817-11e8-8508-899eac4f9c39.png)

Reply:
will leave any for now